src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -105,6 +103,8 @@ private void CheckIfOutputSupportsAlignment()
                     _alignMessages = false;
                 }
             }
+
+            _consoleOutputAligner = new ConsoleOutputAligner(_bufferWidth, _alignMessages);
```

Comment:
nit: `_alignMessages` is not set only in this method. The code seems to be fine (`ApplyParameter` is called before we get here) but still, it is somewhat fragile. I wonder if it wouldn't be better to make `ConsoleOutputAligner` a static class.

Reply:
You are right, it is slightly fragile - good you have binged this up. I have checked code and I believe it is good-enough as it is. `ApplyParameter` has to be called before `CheckIfOutputSupportsAlignment` otherwise `_forceNoAlign` has no proper value.
As for static class `ConsoleOutputAligner` I like it slightly more implemented as it is. This way lifetime of `ConsoleOutputAligner` is controlled by class which uses it `ParallelConsoleLogger`. In particular `ConsoleOutputAligner` instance is recycled and collected after build.